main.js

```diff
@@ -89,3 +89,32 @@ add.addEventListener('click', (e) => {
 });
 
```

Comment:
[OPTIONAL] You can improve the user experience by clearing the input fields for the book title and author after the `Add` button is clicked. This indicates that the book has been successfully added.